app/assets/styles/css/navbar.css

```
@@ -0,0 +1,375 @@
+.btn,
+.navbar .navbar-nav > a.btn {
+  border-width: 2px;
+  font-weight: 400;
+  font-size: 0.8571em;
+  line-height: 1.35em;
+  margin: 5px 1px;
+  border: none;
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-width



(lint/suspicious/noShorthandPropertyOverrides)